src/services/FlowChartServices.ts

```diff
@@ -27,9 +27,44 @@ export function saveFlowChartToLocalStorage(rfInstance?: ReactFlowJsonObject) {
   }
 }
 
-export const sendApiKeyToDjango = async (apiKey: string) => {
+export const sendCloudApiKeyToDjango = async (apiKey: string) => {
   try {
-    const response = await fetch(`${API_URI}/api/set-api`, {
+    const response = await fetch(`${API_URI}/api/set-cloud-api`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ key: apiKey }),
+    });
+
+    if (response.ok) {
+      const responseData = await response.json();
+      notifications.update({
+        id: "set-api-key",
+        title: "Successful!",
```

Comment:
The 3 functions to send the post request are almost exactly the same except for the endpoint they query, please refactor